### PR TITLE
Update to match the new /docs/README.md Triggers Overview from PR 972

### PR DESCRIPTION
**Pull request by sergetron**
# Changes

Updates the top-level project README.md to link to the new Triggers Overview from https://github.com/tektoncd/triggers/pull/972 and remove overlapping content. Also updates the language for clarity and flow.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,78 +8,39 @@
 <img src="tekton-triggers.png" alt="Tekton Triggers logo (Tekton cat playing with a ball)"></img>
 </p>
 
-Triggers is a Kubernetes
+Tekton Triggers is a Kubernetes
 [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
-(CRD) controller that allows you to extract information from events payloads (a
-"trigger") to create Kubernetes resources. If you are looking to start with triggers
-please take a look at the [getting started guide](./docs/getting-started/README.md). 
+(CRD) controller that allows you to create Kubernetes resources based on information it extracts from event payloads called **triggers**.
```
Comment (dibyom) [inline README.md:13]: This makes it sound like the event payloads is called triggers which is confusing.
Maybe we could just remove "called triggers" from this sentence?

Comment (sergetron) [reply README.md:13]: Sure, it sounded like the original version calls them that, so maybe I misunderstood. Removed.

**Pull request merged**